clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClientConfig.scala

```diff
@@ -33,6 +37,17 @@ object ClioClientConfig extends ConfigReaders {
   val serviceAccountJson: Option[Path] =
     clientConfig.getAs[Path]("service-account-json")
 
+  val accessToken: Option[AccessToken] =
+    clientConfig
+      .getAs[String]("access-token")
+      .map(
+        t =>
+          new AccessToken(
+            t,
+            Date.from(Instant.now().plus(Duration.ofMinutes(20)))
+        )
+      )
```

Comment:
Why 20 minutes?

Reply:
Same question here, that seems fishy to hard-code.`null` would be more accurate.